### PR TITLE
[Documentation] tarchive_validation.pl perldocification

### PR DESCRIPTION
**Pull request by cmadjar**
Using perldoc/perlpod to document `tarchive_validation.pl` so that users can type in the terminal
`perldoc tarchive_validation.pl` and get the documentation.

Using the `pod2markdown` tool from perl `Pod::Markdown` module, the `tarchive_validation.pl` file has been created so that we have a web displayed version of the documentation.
`pod2markdown tarchive_validation.pl > tarchive_validation.md`

Dependencies added: perldoc, Pod::Markdown

### CODE REVIEW

File: docs/scripts_md/tarchive_validation.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+\- Verification/determination of the ScannerID of the DICOM study archive
+(optionally creates a new scanner entry in the database if necessary)
+
+\- Optionally, creation of candidates as needed and standardization of gender
+information when creating the candidates (DICOM uses M/F, LORIS database uses
+Male/Female)
+
+\- Check of the CandID/PSCID match. It's possible that the CandID exists, but
+that CandID and PSCID does not correspond to the same candidate. This would
```
Comment (MounaSafiHarab) [inline docs/scripts_md/tarchive_validation.md]: do not instead of "does not"

File: docs/scripts_md/tarchive_validation.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+fail further down silently, so we explicitly check that this information is
+correct here.
+
+\- Validation/Obtention of the SessionID
+
+\- Optionally, completion of extra filtering on the DICOM dataset, if needed
+
+\- Finally, the `isTarchiveValidated` field in the `mri_upload` table is set
+to `TRUE`
```
Comment (MounaSafiHarab) [inline docs/scripts_md/tarchive_validation.md]: I assume set to TRUE if the above conditions/validations were successful

File: docs/scripts_md/tarchive_validation.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The program does the following validations:
+
+\- Verification of the DICOM study archive given as an argument to the script
+using the checksum from the one inserted in the database
+
+\- Verification of the PSC information using whatever field containing the site
+string (typically, the patient name or patient ID)
+
+\- Verification/determination of the ScannerID of the DICOM study archive
```
Comment (MounaSafiHarab) [inline docs/scripts_md/tarchive_validation.md]: @driusan 
do we say "determination" for the act of determining something?
to me I use determination in the context that I am determined to do something :)

File: docs/scripts_md/tarchive_validation.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+\- Optionally, creation of candidates as needed and standardization of gender
+information when creating the candidates (DICOM uses M/F, LORIS database uses
+Male/Female)
+
+\- Check of the CandID/PSCID match. It's possible that the CandID exists, but
+that CandID and PSCID does not correspond to the same candidate. This would
+fail further down silently, so we explicitly check that this information is
+correct here.
+
+\- Validation/Obtention of the SessionID
```
Comment (MounaSafiHarab) [inline docs/scripts_md/tarchive_validation.md]: lol "obtention"; I googled it to make sure it is english (I thought it was a french translation)...
but it looks like it is used oin english, @driusan ?

File: docs/scripts_md/tarchive_validation.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+\-newScanner  : boolean, if set, register new scanners into the database
+
+\-verbose     : boolean, if set, run the script in verbose mode
+
+# DESCRIPTION
+
+The program does the following validations:
+
+\- Verification of the DICOM study archive given as an argument to the script
+using the checksum from the one inserted in the database
```
Comment (MounaSafiHarab) [inline docs/scripts_md/tarchive_validation.md]: I would rephrase this to say:
"Verification of the DICOM study archive given as an argument to the script against the one inserted in the database
using checksum"
or something like that....

File: docs/scripts_md/tarchive_validation.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+\-profile     : name of the config file in ../dicom-archive/.loris-mri
+
+\-reckless    : upload data to the database even if the study protocol
+                is not defined or if it is violated
+
+\-globLocation: loosen the validity check of the tarchive allowing for
+                the possibility that the tarchive was moved to a
+                different directory
+
```
Comment (MounaSafiHarab) [inline docs/scripts_md/tarchive_validation.md]: one minor comment: indentation; see screenshot when I run perldoc at the terminal:
tarchive_validation

**Comment by MounaSafiHarab** [comment]
@cmadjar 
a few minor comments; but tested it and it is all good.
Let me know when you update and I will merge.

**Comment by cmadjar** [comment]
@MounaSafiHarab I integrated all your comments and removed the suspicious words ;)
